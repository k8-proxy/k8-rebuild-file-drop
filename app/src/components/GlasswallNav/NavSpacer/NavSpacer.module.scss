.spacer {
    position: relative;
    width: 100%;
    height: 4.4rem;
    border: none;
    background: transparent;
    font-size: 1.6rem;
    color: #fff;
    text-align: left;
    padding-left: 7rem;
    transition: all 0.3s;
    outline: none;
}