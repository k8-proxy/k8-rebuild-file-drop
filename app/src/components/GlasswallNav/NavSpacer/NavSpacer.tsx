import React from "react";

import styles from "./NavSpacer.module.scss";

const NavSpacer = () => {
    return (
        <li>
            <div className={styles.spacer}></div>
        </li>
    );
};

export { NavSpacer };