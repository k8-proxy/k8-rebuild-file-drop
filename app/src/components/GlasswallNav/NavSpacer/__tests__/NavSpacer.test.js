import React from "react";
import TestRenderer from "react-test-renderer";
import { NavSpacer } from "../NavSpacer";

test("NavSpacer_Snapshot", () => {
    // Arrange
    const component = TestRenderer.create(<NavSpacer />);

    // Act
    let tree = component.toJSON();

    // Assert
    expect(tree).toMatchSnapshot();
});