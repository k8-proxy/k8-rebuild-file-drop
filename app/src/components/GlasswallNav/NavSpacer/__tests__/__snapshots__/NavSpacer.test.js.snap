// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavSpacer_Snapshot 1`] = `
<li>
  <div
    className="spacer"
  />
</li>
`;
